### PR TITLE
[com_installer] Doc Block - copy paste and consistency

### PR DESCRIPTION
**Pull request by brianteeman**
I noticed these copy paste and consistency errors in the foc blocs. If we have doc blocks then they should be usable and correct

### CODE REVIEW

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 798137cab633428f0991cccf11701e1e3217a1ec<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14961">issues.joomla.org/tracker/joomla-cms/14961</a>.</sub>

File: administrator/components/com_installer/models/warnings.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
 defined('_JEXEC') or die;
 
 /**
- * Extension Manager Templates Model
+ *  Installer Warnings Model
```
Comment (Quy) [inline administrator/components/com_installer/models/warnings.php]: Remove extra space before `Installer`.

Comment (brianteeman) [inline administrator/components/com_installer/models/warnings.php]: Thanks

**Review by mbabker — APPROVED** [review]

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c79a331008a0beb8544f44e3b8fb948ce8aa509e<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14961">issues.joomla.org/tracker/joomla-cms/14961</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
@alikon can you please test again?

**Comment by Quy** [comment]
@franz-wohlkoenig No need to test again as only an extra space was removed.

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
thanks

**Pull request merged**